pkgs/development/libraries/yajl/default.nix

```diff
@@ -1,14 +1,14 @@
 { lib, stdenv, fetchFromGitHub, cmake }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
```

Comment:
This isn't used in this commit.

Reply:
Was working on this in parallel with the `composefs` PR and forgot that this one isn't pinned to a tagged commit. Will fix in a minute.